desktop/apps/loyalty/index.js

```diff
@@ -1,4 +1,4 @@
 const express = require('express')
 
 const app = module.exports = express()
-app.use('/loyalty', require('@artsy/reaction-force').default.apps.loyalty)
+app.use('/loyalty', require('@artsy/reaction-force/dist/apps/loyalty/server').default)
```

Comment:
@broskoski Why did you need to make that change btw, did it break for some reason?